flink-table/flink-table-runtime-blink/src/test/java/org/apache/flink/table/runtime/operators/window/WindowOperatorTest.java

```diff
@@ -1045,7 +1045,7 @@ public void testSlidingCountWindow() throws Exception {
 
 		testHarness.processWatermark(new Watermark(12000));
 		testHarness.setProcessingTime(12000L);
-		expectedOutput.add(record("key2", 15L, 5L, 0L));
+		expectedOutput.add(record("key2", 6L, 3L, 0L));
```

Comment:
Do we miss "15L, 5L" here?

Reply:
I think we have not missed outputs here. 
For the slide window(size=5, slide=3), the first window is [-2,2], the second is [1, 5], so here, we only have one output for key2.